src/server/database/Database/Implementation/CharacterDatabase.cpp

```diff
@@ -57,6 +57,8 @@ void CharacterDatabaseConnection::DoPrepareStatements()
     PrepareStatement(CHAR_SEL_CHAR_LEVEL, "SELECT level FROM characters WHERE guid = ?", CONNECTION_SYNCH);
     PrepareStatement(CHAR_SEL_CHAR_ZONE, "SELECT zone FROM characters WHERE guid = ?", CONNECTION_SYNCH);
     PrepareStatement(CHAR_SEL_CHARACTER_NAME_DATA, "SELECT race, class, gender, level FROM characters WHERE guid = ?", CONNECTION_SYNCH);
+    PrepareStatement(CHAR_SEL_CHARACTER_DATA_NAME, "SELECT race, class, gender, name FROM characters WHERE guid = ?", CONNECTION_SYNCH);
```

Comment:
I would merge ```CHAR_SEL_CHARACTER_NAME_DATA``` and ```CHAR_SEL_CHARACTER_DATA_NAME```.
The only additional usage of ```CHAR_SEL_CHARACTER_NAME_DATA``` is in cs_character.cpp:222, so that one has to be changed.